content/en/docs/instrumentation/python/automatic.md

```diff
@@ -28,8 +28,12 @@ with the agent. They should both produce the same results, demonstrating that
 the automatic instrumentation agent does exactly the same thing as manual
 instrumentation.
 
-To better understand auto-instrumentation, see the relevant part of both
-scripts:
+To better understand auto-instrumentation it's useful to know how OpenTelemetry achieves 
+the automatic instrumentation. It happens through monkey-patching which is done the
+[instrumentation libraries][instrumentation]. `opentelemetry-instrumentation-flask` is
+one of these, which we will be using below. 
+
+Now on to an example of manually instrumented code and code that is auto-instrumented
```

Comment:
this sentences confuses me.

Reply:
Well, I wanted to make a segue to the code examples, the prior segue was. 

> To better understand auto-instrumentation, see the relevant part of both scripts:

Happy to take suggestions here.

Reply:
> I would suggest the following copy:

Done